setter.py

```diff
@@ -0,0 +1,36 @@
+from twisted.internet import reactor
+from twisted.python import log
+import sys
+sys.path.insert(0, "kademlia")
+from kademlia.network import Server
+
+log.startLogging(sys.stdout)
+
+if len(sys.argv) != 5:
+    print "Usage: python query.py <bootstrap ip> <bootstrap port> <key> <value>"
+    sys.exit(1)
+
+ip = sys.argv[1]
+port = int(sys.argv[2])
+key = sys.argv[3]
+value = sys.argv[4]
+
+if ":" in value:
+    value = value.split(":")
+print "Getting %s (with bootstrap %s:%i)" % (key, ip, port)
+
+def done(result):
+    pass
+    #reactor.stop()
+
+def bootstrapDone(found, server, key):
+    if len(found) == 0:
+        print "Could not connect to the bootstrap server."
+        reactor.stop()
+    server.set(key, value).addCallback(done)
+
+server = Server()
+server.listen(port-2)
+server.bootstrap([(ip, port)]).addCallback(bootstrapDone, server, key)
+
+reactor.run()
```

Comment:
I added in the query and setter.py files just for testing. They were originally in the demo branch I made. Threw them in to help us test.

As to what this bit of code is doing in particular.
1. It's instantiating the kademlia server node object.
2. It's telling the server to listen on a port. (I should have made it settable from the command line instead of just subtracting the bootstrap port -2)
3. It is giving the server a list of ip/port pairs to bootstrap to. In addition, it is adding a callback. So that once the bootstrap is done, it will attempt to add a key/value pair to the DHT.